src/com/untamedears/PrisonPearl/CombatTagListener.java

```diff
@@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
         if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
             return;
         }
-        UUID plruuid = event.getPlayerUUID();
+        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
```

Comment:
Ah I see what you mean.  Are you running the lastest combat tag?  getPlayerUUID is in the most current source https://github.com/cheddar262/Combat-Tag/blob/master/src/main/java/com/trc202/CombatTagEvents/NpcDespawnEvent.java